### PR TITLE
Folder structure

### PR DESCRIPTION
**Pull request by DushanPramod**
[ITP-199] refactored the project with the given folder structure

[ITP-199]: https://fcodelabs.atlassian.net/browse/ITP-199?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -3,23 +3,31 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { DiaryCardComponent } from './components/diary-card/diary-card.component';
+import { DiaryCardComponent } from './module-1/components/diary-card/diary-card.component';
```
Comment (Dulajdeshan) [inline src/app/app.module.ts]: In the screenshot `module-1` is taken as a sample module to give you an idea. You should use relevant name to your module.

**Review by Dulajdeshan — CHANGES_REQUESTED** [review]
Fix the comments

**Comment by DushanPramod** [comment]
resoleved

File: src/app/app.module.ts (hunk 1 above)
Comment (DushanPramod) [reply src/app/app.module.ts]: resoleved

**Review by Dulajdeshan — APPROVED** [review]

**Pull request merged**